processing/api.py

```diff
@@ -116,7 +129,7 @@ def login(self):
             self._expiry = request_time + 5 * 60 - 30
 
     def _get_jwt(self):
-        url = urljoin(self.api_url, "/authenticate_user")
+        url = urljoin(self.api_url, "api/v1/users/authenticate")
```

Comment:
Is the [refresh token](https://api.cacophony.org.nz/#api-Authentication-RefreshUserAuthentication) handled in using the new authentication api?